### DISCUSSION TITLE
Process finished with exit code 132

### DISCUSSION THREAD

**Discussion by emreozyurek** [airsdk/Adobe-Runtime-Support#1576] [Q&A]
Why I am getting this error on IntelliJ when I am trying to run debug mode? 
I am using SDK 33.1.1.743 on MacOS Monterey. 

(adl quit unexpectedly)

Thank you.

**Comment by ajwfrost** [comment]
Most likely because Monterey sets up a child process with the same entitlements as the parent process i.e. ADL is being launched by IntelliJ and so inherits the permissions from it ... but then, if your application needs to use things like the camera or microphone, it won't be allowed to and will be closed down by the OS because the parent process hadn't declared those entitlements.

We've been looking how to do a workaround for this .. if you launch ADL manually from a terminal window, it should work (or it should ask you about the permissions at least).

thanks

**Comment by emreozyurek** [comment]
Thank you. I will try. 
But this problem does not exist in version 33.1.1.345.
